README.md

```diff
@@ -290,7 +310,10 @@ built from a separate build directory e.g.,
   cmake ../. \
     -DCMAKE_INSTALL_PREFIX="my/install/path/sundials-demo" \
     -DRAJA_BACKEND="SERIAL" \
-    -DENABLE_HDF5="ON"
```

Comment:
Do we want/need a second configuration box that shows the corresponding command for the Docker install, e.g.
```bash
  cd sundials-manyvector-demo
  mkdir build
  cd build
  cmake ../. \
    -DCMAKE_INSTALL_PREFIX="/sundials-demo" \
    -DRAJA_BACKEND="SERIAL" \
    -DENABLE_HDF5="ON"
    -DENABLE_HDF5="ON" \
    -DHDF5_ROOT="/opt/view" \
    -DRAJA_ROOT="/opt/view" \
    -DSUNDIALS_ROOT="/opt/view"
  make
  make install
```